packages/core/src/App/Containers/PasswordSelectionModal/password-selection-modal.jsx

```diff
@@ -28,21 +28,23 @@ const PasswordSelectionModal = observer(
         const { ui } = useStore();
         const { is_mobile } = ui;
 
+        const loggedIn = !!cacheTrackEvents.parseCookies('client_information');
         React.useEffect(() => {
-            trackEventWithCache({
-                name: 'ce_virtual_signup_form',
-                properties: {
-                    action: 'password_screen_opened',
-                    form_name: is_mobile ? 'virtual_signup_web_mobile_default' : 'virtual_signup_web_desktop_default',
+            cacheTrackEvents.pageLoadEvent([
```

Comment:
@aswathy-deriv does this have to be pageLoadEvent or normal event?

Reply:
@agrim-deriv ,we can change it to normal event.The password screen will already have the analytics loaded.Can we try passing the normal event and see in all slow networks and see if the track is happening or not.

Reply:
Sure @aswathy-deriv checking this.

Reply:
Fixed this.